nixnet/session.py

```diff
@@ -33,7 +62,32 @@ def __init__(
             list,
             interface,
             mode):
-        "http://zone.ni.com/reference/en-XX/help/372841N-01/nixnet/nxcreatesession/"
+        """Create an XNET session at run time using strings.
```

Comment:
"using strings" isn't accurate (type wise).  I assume its referring to using named references to database objects rather than receiving database references (in contrast to `nxCreateSessionByRef`).